src/main/java/nz/cri/gns/NZSHM22/util/NZSHM22_ReportPageGen.java

```diff
@@ -204,22 +204,22 @@ public void generateRupSetPage() throws IOException{
 
     public static void main(String[] args) throws IOException {
         NZSHM22_ReportPageGen reportPageGen = new NZSHM22_ReportPageGen();
-       reportPageGen.setName("Min Mag = 8.05")
-       	.setOutputPath("/tmp/reports/m8_V2")
-           .setFillSurfaces(true)
-           .setPlotLevel("DEFAULT")
-//                .setSolution("/home/chrisbc/DEV/GNS/AWS_S3_DATA/WORKING/downloads/SW52ZXJzaW9uU29sdXRpb246MTUzMzYuMExIQkxw/NZSHM22_InversionSolution-QXV0b21hdGlvblRhc2s6NTM3MGN3MmJw.zip");
-   		.setSolution("/tmp/inversions/test_sub_m8.zip");
-       reportPageGen.generatePage();
-
-        /*
-        reportPageGen.setRuptureSet("C:\\Users\\volkertj\\Downloads\\NZSHM22_RuptureSet-UnVwdHVyZUdlbmVyYXRpb25UYXNrOjEwMTA3R2F1Skg=.zip")
+//       reportPageGen.setName("Min Mag = 8.05")
+//       	.setOutputPath("/tmp/reports/m8_V2")
+//           .setFillSurfaces(true)
+//           .setPlotLevel("DEFAULT")
+////                .setSolution("/home/chrisbc/DEV/GNS/AWS_S3_DATA/WORKING/downloads/SW52ZXJzaW9uU29sdXRpb246MTUzMzYuMExIQkxw/NZSHM22_InversionSolution-QXV0b21hdGlvblRhc2s6NTM3MGN3MmJw.zip");
+//   		.setSolution("/tmp/inversions/test_sub_m8.zip");
+//       reportPageGen.generatePage();
+
+
+        reportPageGen.setRuptureSet("C:\\tmp\\NZSHM\\disconnectedPuy.zip")
```

Comment:
@voj any way to avoid having these maybe-broken resources referenced in main methods?? Ideas:

 -  we add them to a DEMO_RESOURCES folder??? unless they're outrageously large ??
 -  or add an instruction on how to create such a file and have a standard subfolder for these that all devs should have ; p[lus add this to .gitignore

Reply:
@chrisbc agree, this has been an ongoing problem. I don't think we can simply add all the zip files that we're referencing. I'll merge this for now and then come up with a general solution.